spec/features/traject/ead2_indexing_spec.rb

```diff
@@ -202,5 +202,27 @@
         end
       end
     end
+
+    describe 'for documents with <acqinfo> elements' do
+      let(:fixture_path) do
+        Arclight::Engine.root.join('spec', 'fixtures', 'ead', 'nlm', 'alphaomegaalpha.xml')
```

Comment:
Since we already have the `ncaids544` fixture commited, can we make an expectation using that fixture on the nested `descrgrp/acqinfo` data in question?